translib/transformer/xlate.go

```diff
@@ -663,9 +642,23 @@ func CallRpcMethod(path string, body []byte, dbs [db.MaxDB]*db.DB) ([]byte, erro
 		if err != nil {
 			return nil, err
 		}
-		ret = data[0].Interface().([]byte)
-		if !data[1].IsNil() {
-			err = data[1].Interface().(error)
+		if len(data) > 0 {
+			if len(data) == RPC_XFMR_RET_ARGS {
+				// rpc xfmr callback returns err as second value in return data list from <xfmr_func>.Call()
+				if data[RPC_XFMR_RET_ERR_INDX].Interface() != nil {
+					err = data[RPC_XFMR_RET_ERR_INDX].Interface().(error)
+					if err != nil {
+						log.Warningf("Transformer function(\"%v\") returned error - %v.", rpcFunc, err)
```

Comment:
Here and elsewhere, please us log.V(lvl).Infof(...)

Reply:
Prefer to let it it be a warning since its kind of error returned. Putting it under verbosity level might hide the error if verbosity enabled/set.